SECURITY-INSIGHTS.yml

```
@@ -0,0 +1,66 @@
+header:
+  schema-version: 1.0.0
+  expiration-date: '2025-02-13T10:10:09.000Z'
+  last-updated: '2024-02-13'
+  last-reviewed: '2024-02-13'
+  commit-hash: 4dbf78ebc006ee5f668c0a74876ef8d6db9485be
+  project-url: https://github.com/ossf/S2C2F-attestation-schema-and-tool
+  project-release: '0.0.0'
+  changelog: https://github.com/ossf/S2C2F-attestation-schema-and-tool/changelog.md
+  license: https://github.com/ossf/S2C2F-attestation-schema-and-tool/blob/main/LICENSE
+project-lifecycle:
+  status: active
+  roadmap: https://github.com/ossf/S2C2F-attestation-schema-and-tool/roadmap.md
+  bug-fixes-only: false
+  core-maintainers:
+  - https://github.com/adriandiglio
+  - https://github.com/camaleon2016
+  - https://github.com/yogitasrivastava
+  release-cycle: https://github.com/ossf/S2C2F-attestation-schema-and-tool/releases
+  release-process: To be planned for release as a GitHub Action, and published into the GitHub Marketplace.
+  status: WIP
+contribution-policy:
+  accepts-pull-requests: true
+  accepts-automated-pull-requests: true
+  automated-tools-list:
+  - automated-tool: Dependabot
+    action: allowed
+  contributing-policy: https://github.com/ossf/S2C2F-attestation-schema-and-tool/contributing.md
```

Also need to include branch in URL here or it will error out.

```suggestion
  contributing-policy: https://github.com/ossf/S2C2F-attestation-schema-and-tool/blob/main/contributing.md
```